### ISSUE TITLE
[Question] Vanilla (?) internal cam behavior

### ISSUE THREAD

**Issue by ajsglist** [Zaggy1024/LC_OpenBodyCams#50]
Hey there,

I apologize in advance for the inconvenience.  I had a question about the vanilla internal ship cam.  I thought you might have some insight given OBC and your time spent with the game.  Our group has the internal camera shifted to the large monitor when OBC hasn't been purchased.  We recently realized that when players look up/down, a second set of arms is visible.  While very funny, I was curious if you knew what was going on here.  Is this just vanilla behavior with displaying the player model at the same time as the first person hands model?

And beyond that, do you know of any way to fix this?  We looked around but couldn't seem to find anything.  Obviously this is outside the scope of OBC, but you seemed to be the best person to ask since we couldn't make heads or tails of this.

Cheers

![image](https://github.com/user-attachments/assets/0055502f-2991-4633-84c2-0c500d913296)
![image](https://github.com/user-attachments/assets/bd18c7e5-6b51-4c7c-86fd-1b0ae73fceb5)

**Comment by Zaggy1024** [comment]
Yeah, unfortunately this is a vanilla bug. The setup in the vanilla game is a bit weird honestly, the way that the player model is visible in that camera is by restricting the LOD to the lowest quality one, while the highest quality player model is hidden for the local player. The player arms are also in the default layer, so they can't be hidden without changing that or using the system that OpenBodyCams does to override object visibility per camera.

I think I remember there being some mods that move the hands to a different layer so they can be hidden in the internal camera. However, the ones I thought did so don't actually appear to still do what I remember. Regardless, I wouldn't tend to say that should be the solution, since iirc the solutions that people tend to use result in enemies that are intended to be invisible becoming visible on cameras that need to see the third-person model. (It's been a long time since I had to look at this stuff.)

I could use the camera callbacks I have set up in OpenBodyCams to just override the visibility for the internal camera instead, which would solve this problem and should hopefully also fix the visibility of cosmetics if they aren't appearing there.
